demo/snapshot-only-plugins.txt

```diff
@@ -1 +0,0 @@
-pipeline-milestone-step
```

Comment:
BTW I intend to rewrite how plugins are selected to match the much superior system in https://github.com/jenkinsci/parallel-test-executor-plugin/pull/20.